Start button bug

bee25141 opened the pull request:
Added active/inactive states to the start and stop buttons to prevent starting duplicate instances

bee25141 committed:
working on getting state to properly change on start click

bee25141 committed:
state of start button switches when clicked, and does not start another time instance when clicked after instance currently running

bee25141 committed:
Able to change the state of buttons

No longer allows for multiple time instances to be created when one in progress

bee25141 committed:
Merge branch 'master' into start-button-bug

bee25141 committed:
Cleaned up debuggers on page

bee25141 committed:
Merge branch 'master' into start-button-bug

bee25141 committed:
Removed page refresh button

Erroneously appeared again on master; removing and replacing with signInDisplay function

bee25141 committed:
Merge branch 'master' into start-button-bug

bee25141 committed:
Merge branch 'master' into start-button-bug

bee25141 committed:
Merge branch 'master' into start-button-bug

burke1791 reviewed (changes requested):
I like what you're going for here, but there's one thing that might cause problems down the line. 

In the html page, you're trying to give both buttons two state attributes.  This can't happen because both attributes are named "state" so the browser will default to the first "state" attribute it sees, in this case the inactive state.  I would remove the initial active state from each button in index.html to avoid confusion.

bee25141 committed:
Corrected states in html

made only one state for button; states toggle between 'active' and 'inactive' upon click

bee25141 commented:
Thanks for heads up! Had a feeling that wasn't the best way to go about it but initially had trouble. Should be good to go now, only one state and it toggles between 'active' and 'inactive' upon click. If not let me know and will revise!

burke1791 reviewed (approved):
Nice work.  Thanks for putting up with my overly picky comments.
